### PR TITLE
Dala 3415 provide additional information during sign on

### PR DESCRIPTION
**Pull request by marcelrasbachdfine**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [ ] At least one test exists testing the new feature
  - [ ] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] At least 3 consecutive CI runs are successfully executed to ensure that there are no flaky tests.
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] A fresh clone of dataland.com is generated (see Wiki page on "OTC" for details)
  - [ ] The feature branch is deployed to clone with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the clone server
  - [ ] The feature branch is deployed to dev1 with `Reset non-user related Docker Volumes & Re-populate` turned on, and it's verified that the CD run is green  
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [x] The new feature is manually used/tested/observed on dev server

### CODE REVIEW

File: dataland-keycloak/dataland_theme/login/templates/register.ftl [hunk 1]
```diff
@@ -64,6 +95,22 @@
...
+            let submit_button = document.getElementById("register_button");
+
+            let first_name_field = document.getElementById("firstName");
+            let last_name_field = document.getElementById("lastName");
+
+            submit_button.addEventListener("click", function () {
+                console.log("changing names");
+                if(!first_name_field.value) first_name_field.value = "-";
+                if(!last_name_field.value) last_name_field.value = "-";
+                console.log("first name: " + first_name_field.value);
```
Comment (aliabdoli77) [inline dataland-keycloak/dataland_theme/login/templates/register.ftl:109]: please remover logger if unnecessary

File: dataland-keycloak-adapter/src/main/kotlin/org/dataland/keycloakAdapter/auth/DatalandJwtAuthentication.kt [hunk 2]
```diff
@@ -14,6 +14,12 @@ class DatalandJwtAuthentication(private val jwt: Jwt) : DatalandAuthentication()
     val username: String
         get() = jwt.getClaimAsString("preferred_username")
 
+    val firstName: String
+        get() = jwt.getClaimAsString("given_name") ?: "" // todo check if this is best practise
```
Comment (aliabdoli77) [inline dataland-keycloak-adapter/src/main/kotlin/org/dataland/keycloakAdapter/auth/DatalandJwtAuthentication.kt:18]: remove todo

File: dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/messaging/DataOwnershipEmailMessageSender.kt [hunk 3]
```diff
@@ -46,6 +52,7 @@ class DataOwnershipEmailMessageSender(
             "Data Ownership Request",
             properties,
         )
+        logger.info(objectMapper.writeValueAsString(message))
```
Comment (aliabdoli77) [inline dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/messaging/DataOwnershipEmailMessageSender.kt:55]: remove logger and import

Comment (marcelrasbachdfine) [reply dataland-backend/src/main/kotlin/org/dataland/datalandbackend/services/messaging/DataOwnershipEmailMessageSender.kt:55]: done

File: dataland-keycloak/dataland_theme/login/templates/register.ftl (hunk 1 above)
Comment (marcelrasbachdfine) [reply dataland-keycloak/dataland_theme/login/templates/register.ftl]: in my version of the code, the logger doesnt exist (anymore) but even before addressing your comment

Comment (stewal19) [reply dataland-keycloak/dataland_theme/login/templates/register.ftl]: Is no longer there

**Pull request merged**